rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+|-------------------------------------------------------	|--------------------------	|-----------------	|---------------	|
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis Pipelines                                    	| Broad                    	| Q2 2019         	| ?             	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| TBD             	| S. Ehsan      	|
+| Data Analysis Service starfish Pipeline Tests         	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| ?             	|
+| [Ability to describe files that belong together][a]   	| ?                        	| ?               	| ?             	|
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| ?             	|
+| DSS spatial data bundle (or bundless) specification   	| UCSC                     	| ?               	| B. Hannafious 	|
+| DSS scale testing                                     	| UCSC / DCP               	| TBD             	| B. Hannafious 	|
+| Data Browser facets for spatial data                  	| UCSC                     	| TBD             	| H. Schmidt    	|
+| Matrix service integration                            	| starfish                 	| TBD             	| M. Kinsella   	|
+| Upload Bundle for 1M+ files                           	| ?                        	| ?               	| P. Shah       	|
+| Image file download latency testing                   	| ?                        	| ?               	| ?             	|
+
+### **Unresolved Questions**
+
+  - Who are the intended beta users for imaging support and what
+    functionality is being targeted?
+
+  - Are there other roadblocks for imaging support in the DCP?
+
+  - What, if any, additional automation may be needed in the DCP to
+    support imaging?
```

Comment:
Support (tech and wrangling) for contributors outside of the SpaceTx project to convert to this format or support for ingestion of other formats.

Reply:
@joshmoore can you add a brief section that covers our intentions with respect to Bio-Formats?

Reply:
@hewgreen : let me know what you think about 6d3eabfbb0b9f06a97d405a6bdb2ced00b3b589c -- i.e. added as a drawback with existing efforts for remediation.

Reply:
Thanks Josh, looks good. At this stage predicting this requirement is sufficient because it isn't clear if this will be frequently required, prove too difficult or produce an interoperable result.